scl/rewrite/cc-mask.conf

```diff
@@ -38,7 +38,14 @@
 # A notable difference compared to the blog post, is that the hash_cc
 # rule is called credit-card-hash and mask_cc is credit-card-mask.
 
-@define balabit.credit-card-regexp "(:4[0-9]{12}(?:[0-9]{3})?|5[1-5][0-9]{14}|6(?:011|5[0-9][0-9])[0-9]{12}|3[47][0-9]{13}|3(?:0[0-5]|[68][0-9])[0-9]{11}|(?:2131|1800|35\d{3})\d{11})"
+@define balabit.credit-card-regexp.visa "(?:4[0-9]{12}(?:[0-9]{3})?)"
```

Comment:
Could we please add some test covering at least all credit card format here ? (preferably light tests)